frontend/src/pages/protected/SelectBot.tsx

```
@@ -0,0 +1,26 @@
+function SelectBot() {
+	return (
+		<div className="select-bot-interface-container">
+			<DashboardNavbar />
+			<div className="main-page-content">
+				<h1 className="select-bot-page-heading">Play vs bot</h1>
+				<p className="select-bot-page-description">
+                    Play against a bot of your choice
+                </p>
+                <div className="bots-container">
+					{botsTable.map((categoryInfo, _) => (
+						<BotCategory categoryInfo={categoryInfo} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)